aquadoggo/src/lib.rs

```diff
@@ -25,3 +25,23 @@ mod test_helpers;
 
 pub use config::Configuration;
 pub use runtime::Runtime;
+
+/// Init pretty_env_logger before the test suite runs to handle logging outputs.
+///
+/// Several of our dependencies (`sqlx`, `p2panda_rs`, `tide`) emmit log messages
+/// which we can handle and print using `pretty_env_logger`. Logging ehaviour
+/// can be customised at runtime. With eg. `RUST_LOG=p2panda_rs=info cargo t` or
+/// `RUST_LOG=debug cargo t`.
+///
+/// The `ctor` crate is used to define a global constructor function. This method
+/// will be run before any of the test suites.
```

Comment:
Nice writeup!

Can see some typos here and there ("emmit", "ehaviour") :broccoli: